packages/api-clients/open-api/purposeApi.yml

```diff
@@ -1606,10 +1607,11 @@ components:
           minLength: 5
           maxLength: 60
         riskAnalysisForm:
-          $ref: "#/components/schemas/RiskAnalysisFormSeed"
           description: |
             Optional in the purpose model, but a purpose cannot exist without a risk analysis.
             There is no practical use in letting the user remove it, we don't make it nullable.
+          allOf:
+            - $ref: "#/components/schemas/RiskAnalysisFormSeed"
```

Comment:
Is this change correct?
This `$ref` seemed correct,  while now we have `allOf:` with a single reference below...
Same for other occurrences of this similar change